.github/workflows/release.yml

```diff
@@ -16,7 +16,7 @@ on:
 
 env:
   JAVA_RELEASE: '23'
```

Comment:
Only one of `JAVA_RELEASE` or `JAVA_VERSION` is effectively needed when we decide to use the latest version. But this could be another PR.